packages/protocol/test-sol/unit/governance/network/Governance.t.sol

```diff
@@ -3088,7 +3205,7 @@ contract GovernanceTest_whitelistHotfix is GovernanceTest {
     governance.whitelistHotfix(HOTFIX_HASH);
   }
 
-  function test_Reverts_WhenCalledOnL2() public {
+  function test_Reverts_WhenL2() public {
```

Comment:
Consider moving this out to a separate GovernanceTest_whitelistHotfix_L2 test contract for better organization.